web/app/scripts/src/services/calculate-percentage.es.js

```diff
@@ -15,12 +15,17 @@ const module = angular.module(moduleProperties.moduleName, []);
 
 module.service(moduleProperties.componentName, [
     function () {
+        let qKeys = [];
```

Comment:
It seems a bit risky to have `qKeys` defined like this, storing state on the service, outside the methods `countQuestionsTotal` and `countNumberOfAnsweredQuestions`